packages/aws-lambda/src/utils/version.ts

```diff
@@ -0,0 +1,2 @@
+export const awsLamdaTypesVersion = '8.10.92';
+export const esbuildVersion = '^0.17.5';
```

Comment:
The `esbuildVersion` is not needed right?

Reply:
it's needed